packages/conf-adwaita-icon-theme/conf-adwaita-icon-theme.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "7895506+MSoegtropIMC@users.noreply.github.com"
+homepage: "https://github.com/adwaita/adwaita-icon-theme"
+bug-reports: "https://gitlab.adwaita.org/adwaita/adwaita-icon-theme/issues"
+authors: "GNU Project"
+license: ["LGPL-3.0-only" "CC-BY-SA-3.0"]
+# E.g. on Fedora the system package does not contain a .pc file.
+# Testing for this folder seems to be more portable than using system packager commands as fallback.
+build: [ "sh" "-c" "pkg-config --short-errors --print-errors adwaita-icon-theme || [ -d /usr/share/icons/Adwaita ]"]
```

Comment:
Neat! Does `[ -d /usr/share/icons/Adwaita ]` work also on `sh` (when it is not bash)?

Reply:
Well I call "sh" explicitly here, so it should be sh and not bash - the right part is tested on Fedora and works. But to make sure I started a sh and tried it and it seems to work.